File: BasicDemoApplication/app/src/main/java/com/webinstats/basicdemoapplication/MainActivity.java
package com.webinstats.basicdemoapplication;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import java.util.HashMap;
import java.util.Map;

import webinstats.android_wis.Webinstats;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Webinstats wiso = new Webinstats("//__YOUR__DOMAIN__.webinstats.com/","__COMPANY__ID__","0");
        Map<String,String> map=new HashMap<>();
        map.put("p","Other");
        map.put("_enable_push","1");
        // call execute function on onCreate() method
        wiso.execute(this,map);

    }
}

File: KotlinDemoProject/app/src/main/java/com/webinstats/kotlindemoproject/MainActivity.kt
package com.webinstats.kotlindemoproject

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import webinstats.android_wis.Webinstats

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val wiso = Webinstats("//__YOUR_DOMAIN_.webinstats.com/","__YOUR_COMPAY_ID__","0")
        val map= HashMap<String,String>()
        map.put("p","Other")


        wiso.execute(this,map)


    }
}
